compiler/src/iree/compiler/Codegen/SPIRV/test/tile_and_distribute.mlir

```diff
@@ -267,10 +267,9 @@ hal.executable private @conv_2d {
 //         CHECK:             scf.for %[[IV_IC:.+]] = %[[C0]] to %{{.+}} step %[[C4]]
 //         CHECK:               %[[INPUT:.+]] = memref.subview %[[INPUT_THREAD]]
 //         CHECK:               %[[FILTER:.+]] = memref.subview %[[FILTER_THREAD]]
-//         CHECK:               %[[OUTPUT_SUBVIEW:.+]] = memref.subview %[[OUTPUT]]
 //         CHECK:               linalg.conv_2d_nhwc_hwcf
 //    CHECK-SAME:                 ins(%[[INPUT]], %[[FILTER]]
-//    CHECK-SAME:                 outs(%[[OUTPUT_SUBVIEW]]
+//    CHECK-SAME:                 outs(%[[OUTPUT]]
```

Comment:
Okay, this is changed back by the PR. This is fine, see explanation in https://github.com/openxla/iree/pull/15869#issue-2033521799